vm/systemSmartContracts/auction.go

```diff
@@ -724,6 +726,63 @@ func checkDoubleBLSKeys(blsKeys [][]byte) bool {
 	return false
 }
 
+func (s *stakingAuctionSC) cleanRegisteredData(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if !s.flagDoubleKey.IsSet() {
+		s.eei.AddReturnMessage("invalid method to call")
+		return vmcommon.UserError
+	}
+
+	err := s.eei.UseGas(s.gasCost.MetaChainSystemSCsCost.Stake)
```

Comment:
So the cost for cleaning is s.gasCost.MetaChainSystemSCsCost.Stake?

Reply:
yes.